vendor/github.com/brocaar/lorawan/band/band_au915_928.go

```diff
@@ -0,0 +1,186 @@
+// +build au_915_928
+
+package band
+
+import (
+	"fmt"
+	"time"
+)
+
+func init() {
+	// initialize uplink channel 0 - 63
+	for i := 0; i < 64; i++ {
+		UplinkChannelConfiguration[i] = Channel{
+			Frequency: 915200000 + (i * 200000),
+			DataRates: []int{0, 1, 2, 3},
+		}
+	}
+
+	// initialize uplink channel 64 - 71
+	for i := 0; i < 8; i++ {
+		UplinkChannelConfiguration[i+64] = Channel{
+			Frequency: 915900000 + (i * 1600000),
+			DataRates: []int{4},
+		}
+	}
+
+	// initialize downlink channel 0 - 7
+	// In Australia this is set from 10-13 if we reside in offset 0
+	// Mentioned in
+	// LoRaWAN 1.0. Standard - Section 7.5.7
+	for i := 0; i < 8; i++ {
+		DownlinkChannelConfiguration[i] = Channel{
+			Frequency: 923300000 + (i * 600000),
+			DataRates: []int{10, 11, 12, 13},
```

Comment:
This should be DR8-DR13 I think. The offset value will be derrived from ``RX1DROffsetConfiguration``.